src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -74,11 +78,13 @@
       </ul>
     {/if}
   </div>
-  <div class="bg-[#DDD6FE]">
+  <div
+    class="flex h-64 items-center justify-center overflow-hidden bg-[#DDD6FE] xl:h-auto xl:flex-1"
```

Comment:
I think the flex broke some things here ⬇️  

| Before | After |
|-|-|
|<img width="1093" height="913" alt="Screenshot 2025-07-10 at 2 12 13 PM" src="https://github.com/user-attachments/assets/ec1aacb3-fdd1-4c81-b1e3-2797711e82e6" />|<img width="1085" height="709" alt="Screenshot 2025-07-10 at 2 13 29 PM" src="https://github.com/user-attachments/assets/ddd789bb-56bf-4c43-8987-b674b7d91cd9" />|

I'd leave the rest of the styling as is for now and just fix the light vs. dark image.